lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -503,6 +504,7 @@ def _verify_dl_flow(self, dl_flow_rules=None):
             dst_addr = "nw_dst" if key.version == 4 else "ipv6_dst"
             key_to_be_matched = "ipv4_src" if key.version == 4 else "ipv6_src"
             eth_typ = 2048 if key.version == 4 else 34525
+            in_port = self.LOCAL_PORT_NON_NAT_IPV6 if ipv6_non_nat else self.LOCAL_PORT
```

Comment:
Can IPv4 and IPv6 flow rules co-exist? if yes shall we not need to verify both of them together?

Reply:
It can co-exist. But the verification happens one at a time in the order the ip addresses are passed to this function